source/common/http/http1/conn_pool.cc

```diff
@@ -310,7 +310,7 @@ void ConnPoolImpl::StreamWrapper::decodeHeaders(HeaderMapPtr&& headers, bool end
       saw_close_header_ = true;
       parent_.parent_.host_->cluster().stats().upstream_cx_close_notify_.inc();
     } else if (absl::EqualsIgnoreCase(headers->Connection()->value().getStringView(),
-                               Headers::get().ConnectionValues.KeepAlive)) {
+                                      Headers::get().ConnectionValues.KeepAlive)) {
```

Comment:
Revert?

Reply:
This fixes the formatting.

Reply:
Yes, this is going to be merged upstream.